Providers/Scripts/2.4x-2.5x/Scripts/nxOMSSyslog.py

```diff
@@ -19,10 +19,17 @@
 oms_syslog_ng_conf_path = '/etc/opt/omi/conf/omsconfig/syslog-ng-oms.conf'
 oms_rsyslog_conf_path = '/etc/opt/omi/conf/omsconfig/rsyslog-oms.conf'
 conf_path = ''
+default_port = '25224'
```

Comment:
Since the old version of the module didn't read it, I don't want to create a scenario where it can't be read and we fail because of that. Perhaps there is some old version of the agent which doesn't store the port in one of the two locations that we expect.